AEPCore/Sources/eventhub/EventSource.swift

```diff
@@ -36,4 +36,8 @@ public class EventSource: NSObject {
     public static let applicationClose = "com.adobe.eventSource.applicationClose"
     public static let personalizationDecisions = "personalization:decisions"
     public static let locationHintResult = "locationHint:result"
+    public static let errorResponseContent = "com.adobe.eventSource.errorResponseContent";
+    public static let requestTracker = "com.adobe.eventSource.requestTracker";
+    public static let trackMedia = "com.adobe.eventSource.trackMedia";
+    public static let sessionCreated = "com.adobe.eventSource.sessionCreated";
```

Comment:
We previously dispatched a similar event in media to match client API events with hitdebugger response. Does this still apply to edge media?

Reply:
We have kept this for now. We don't have the integration with hitdebugger for now, but I am thinking we might need this for low level events validation in griffon.

Reply:
@addb there will likely be no client-side integration with hitdebugger in the case of Edge Media. For debugging purposes can we relay exclusively on the `media-analytics:new-session` response handle and not dispatch this additional internal event?